addons/calendar/models/calendar_event.py

```diff
@@ -24,6 +31,12 @@
     'start_date': 'sort_start',
 }
 
+def get_weekday_index(weekday):
+    """
```

Comment:
should this method be public?

Reply:
`get_weekday_index` is not a method, it's just a function so I guess we cannot call it through a RPC call, isn't it ?